=== modified file 'CMakeLists.txt'
--- CMakeLists.txt	2012-05-21 20:16:56 +0000
+++ CMakeLists.txt	2013-01-04 10:48:09 +0000
@@ -48,8 +48,8 @@
 if (Qt5Widgets_FOUND)
    find_package(Qt5DBus REQUIRED)
    include_directories(${Qt5Widgets_INCLUDE_DIRS} ${Qt5DBus_INCLUDE_DIRS})
-   find_package(extra-cmake-modules 0.0.2 REQUIRED)
-   set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${EXTRA_CMAKE_MODULES_MODULE_PATH})
+   find_package(ECM 0.0.4 REQUIRED)
+   set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${ECM_MODULE_PATH})
    find_package(Qt5Transitional REQUIRED Core)
    set(CMAKE_AUTOMOC ON)
    set(CMAKE_AUTOMOC_RELAXED_MODE ON)

